admin-ui/src/pages/request-detail-page.tsx

```
@@ -230,6 +230,40 @@ export function RequestDetailPage(): React.JSX.Element {
                     )}
                   </TableCell>
                 </TableRow>
+                <TableRow>
+                  <TableCell className="text-muted-foreground">user_id</TableCell>
```

ESLint

[error] Replace `user_id` with `⏎····················user_id⏎··················`

(prettier/prettier)